api/src/main/resources/workbench.yaml

```diff
@@ -2974,7 +2975,17 @@ definitions:
         type: integer
         format: int64
       resourceType:
-        type: string
+        $ref: "#/definitions/ResourceType"
+
+  ResourceType:
```

Comment:
That's true. In this case, I decided to keep `Type` since that was the existing naming convention we were using across the front end and I did felt like it added a little bit of meaning.